src/utils/generator/EqualVelocityAssigner.h

```diff
@@ -5,19 +5,19 @@
 
 #include "VelocityAssignerBase.h"
 
-/** The VelocityAssigner can be used to assign normally distributed velocity vectors with absolute value mathching to a given temperature.
+/** The VelocityAssigner can be used to assign normally distributed velocity vectors with absolute value matching a given temperature.
  */
 class EqualVelocityAssigner : public VelocityAssignerBase {
 public:
-	EqualVelocityAssigner(double T = 0, int seed = 0) : VelocityAssignerBase(T), _mt(seed), _uniformDistrubtion(0, 1) {}
+	EqualVelocityAssigner(double T = 0, int seed = 0) : VelocityAssignerBase(T), _mt(seed), _uniformDistribution(0, 1) {}
 	~EqualVelocityAssigner(){}
 
 	void assignVelocity(Molecule *molecule) {
-		double v_abs = sqrt(/*kB=1*/ T() / molecule->component()->m());
+		double v_abs = sqrt(/*kB=1*/ (3+molecule->component()->getRotationalDegreesOfFreedom())*T() / molecule->component()->m());
```

Comment:
should this then also not be included?

Reply:
This must be included! In the EqualVA, the old calculation of the absolute velocity is wrong.
Summarizing:
EqualVA: Typos and fix of v_abs calculation
MaxwellVA: Only typos

Reply:
why would the rotational degrees of freedom be used in only one of the velocityassigners, but not in both?
please give me a physical explanation ;)

Reply:
Sorry, you are right! Of course, the rotational degrees of freedom must be considered in both cases, since the calculation of the global temperature is more or less: T = Ekin/DOF with DOF being both the translational and rotational degrees of freedom.
I added a commit (16c4ac2) in which the calc of the abs velo was also fixed for the Maxwell VA.